arch/x86/soc/ia32/soc.c

```diff
@@ -26,4 +26,9 @@ MMU_BOOT_REGION(CONFIG_IOAPIC_BASE_ADDRESS, MB(1), MMU_ENTRY_WRITE);
 #ifdef CONFIG_HPET_TIMER
 MMU_BOOT_REGION(CONFIG_HPET_TIMER_BASE_ADDRESS, KB(4), MMU_ENTRY_WRITE);
 #endif
+
+#ifdef CONFIG_ETH_E1000
+MMU_BOOT_REGION(CONFIG_ETH_E1000_BASE_ADDRESS, KB(128), MMU_ENTRY_WRITE);
```

Comment:
DTS [here](https://github.com/zephyrproject-rtos/zephyr/pull/9112/commits/6e7b16c1e47ffb65b22105e2826155146ef012b4#diff-8c50a13bfd9cee1a815504f63ecc98e0R34) specifies that there're just 0x100 bytes of registers, while here 128K is reserved. Is this correct?

Reply:
Also, ideally this would have been a separate commit...